querqy-core/src/main/java/querqy/rewrite/commonrules/LineParser.java

```diff
@@ -161,6 +170,32 @@ public static Object parse(final String line, final Input previousInput,
 
     }
 
+    protected static RawQuery parseRawQuery(final String rawQuery, final Clause.Occur occur) throws RuleParseException {
+        final List<String> rawQueryParts = Arrays.asList(rawQuery.split("%%", -1));
+
+        final int size = rawQueryParts.size();
+
+        if (size % 2 == 0) {
+            throw new RuleParseException("Invalid use of parametrization in the definition of a RawQuery. " +
+                    "Parameters must begin and end with %%");
+        }
+
+        if (size == 1) {
+            return new StringRawQuery(null, rawQuery, occur, false);
+        } else {
+            final List<ParametrizedRawQuery.Part> parts = IntStream.range(0, size)
+                    .mapToObj(index -> index % 2 == 0
+                            ? new ParametrizedRawQuery.Part(rawQueryParts.get(index),
+                                ParametrizedRawQuery.Part.Type.QUERY_PART)
+                            : new ParametrizedRawQuery.Part(rawQueryParts.get(index).trim(),
+                                ParametrizedRawQuery.Part.Type.PARAMETER))
+                    .filter(part -> !part.isEmpty())
```

Comment:
I'm 99.9% sure we do not want empty parts but what if someone needs it. I guess it wouldn't hurt to leave them in?!